jersey-servers/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile "com.fasterxml.jackson.module:jackson-module-afterburner"
     compile "com.jcraft:jzlib"
     compile "com.palantir.safe-logging:safe-logging"
+    compile "com.palantir.tokens:auth-tokens"
```

Comment:
yeah, locally moved over but then hit the blocker that there's not an ErrorType for a 401.

Will at some point add code instead to auth tokens, not as param converter though. There's no (non deprecated) remoting support any more for throwing an exception that'll get mapped into the right thing, so instead I'll have to make a filter